package.json

```diff
@@ -48,7 +48,9 @@
     "publishCmd": "npm publish",
     "publishDevCmd": "npm publish --tag ${devTag}",
     "versionCmd": "npm version --no-git-tag-version ${version}",
-    "cleanCmd": "git checkout -- package.json",
+    "checkFilesTracking": "git ls-files --error-unmatch ${filesToClean}",
```

Comment:
this only deals with a single file, so you probably want to take in the file and not filesToClean. Also the name should reflect that it only checks a single file at a time.